addons/web/static/tests/qunit.js

```diff
@@ -398,8 +406,12 @@ export function setupQUnit() {
     QUnit.begin(function () {
         if (odoo.__DEBUG__.services["@web/core/errors/error_utils"]) {
             const errorUtils = odoo.__DEBUG__.services["@web/core/errors/error_utils"];
```

Comment:
i guess there is a reason for that, but why is this not imported at the top of the file, with an `import`, like everything else?